common/scylla_configure.py

```diff
@@ -78,6 +78,9 @@ def updated_ami_conf_defaults(self):
         self.CONF_DEFAULTS["scylla_yaml"]["broadcast_rpc_address"] = private_ip
         self.CONF_DEFAULTS["scylla_yaml"]["seed_provider"][0]['parameters'][0]['seeds'] = private_ip
         self.CONF_DEFAULTS["scylla_yaml"]["endpoint_snitch"] = self.cloud_instance.endpoint_snitch
+        stream_bandw = estimate_streaming_bandwidth()
+        if stream_bandw != 0:
+            self.CONF_DEFAULTS["scylla_yaml"]["stream_io_throughput_mb_per_sec"] = stream_bandw
```

Comment:
```suggestion
self.CONF_DEFAULTS["scylla_yaml"]["stream_io_throughput_mb_per_sec"] = estimate_streaming_bandwidth()
```
No need to special case zero as it is the default anyhow and means no limit.

Reply:
But setting it implies we've figured out a reasonable value. Which we have not. I prefer to leave it up to scylla to decide, should default change from 0 in the future.

Reply:
The confusion is in the gap between `estimate_streaming_bandwidth`'s API and the the configuration is being used for.
In the function it means "couldn't figure out" .
In the configuration it is "no limit".

But fine, leave it with the `if`, giving a chance to "translate" the function's return to something else.